src/shared/prerendered-app/Intro/style.css

```
@@ -118,7 +118,115 @@
   fill: var(--light-blue);
 }
 
+.bottom-wave {
+  position: relative;
+}
+
+.info {
+  background: var(--white);
+  position: relative;
+  padding: 5em 2em;
+  &:nth-of-type(even) {
+    & .info-img-wrapper {
+      order: 1;
+      @media (min-width: 768px) {
+        justify-self: start;
+      }
+    }
+
+    & .info-text-wrapper {
+      order: 2;
+      text-align: right;
```

```suggestion
      text-align: right;
      align-items: flex-end;
```

fixes the link hit/focus area from being the width of the container